plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -3484,6 +3483,34 @@ eosio::chain::backing_store_type read_only::get_backing_store() const {
 }
 
 } // namespace chain_apis
+
+fc::variant chain_plugin::get_entire_trx_trace(const transaction_trace_ptr& trx_trace )const {
+
+    fc::variant pretty_output;
+    auto ro_api = get_read_only_api();
```

Comment:
I think you should modify `make_resolver` to take a `controller&` or `chainbase&` so you don't have to make a copy of `read_only` on every call.

Reply:
Fixed.